ambry-account/src/main/java/com/github/ambry/account/HelixAccountService.java

```diff
@@ -246,6 +250,69 @@ public boolean updateAccounts(Collection<Account> accounts) {
     return updateAccountsWithAccountMetadataStore(accounts, accountMetadataStore);
   }
 
+  /**
+   * {@inheritDoc}
+   */
+  @Override
+  public Pair<String, String> addContainer(String accountName, Container newContainer) {
```

Comment:
This entire method is generic except for the line that updates the underlying store.  Can we move it to the base class?  Maybe AccountService can also have a method like getMetadataStore().

Reply:
Good point. The only blocker here is how to pass in a config to specify "container id start number". Or can we hard code it somewhere? (I am thinking if container id can start from 0)

Reply:
Since you added it to config, can you make the method generic now?

Reply:
Sure, will do